app/models/exercise.rb

```diff
@@ -27,6 +27,8 @@ class Exercise < ActiveRecord::Base
     where(course_id: course.id, gdocs_sheet: gdocs_sheet)
   }
 
+  enum disabled_status: [:enabled, :disabled]
```

Comment:
Jamo wanted an enum field for disabling courses (https://github.com/testmycode/tmc-server/pull/265#discussion_r31899442) so I thought why not do the same thing with disabling exericses?